File: AVc/app/src/main/java/com/hfad/avc/ui/database/AppDatabase.java
package com.hfad.avc.ui.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {Contact.class,Template.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract ContactDao contactDao();
    public abstract TemplateDao templateDao();
}

File: app/src/main/java/com/hfad/avc/data/database/AppDatabase.kt
package com.hfad.avc.data.database

import androidx.room.Database
import androidx.room.RoomDatabase
import androidx.room.TypeConverters
import com.hfad.avc.Util.Converter
import com.hfad.avc.data.model.Contact
import com.hfad.avc.data.model.Event
import com.hfad.avc.data.model.Template

@Database(entities = [Contact::class, Template::class, Event::class], version = 1)
@TypeConverters(Converter::class)
abstract class AppDatabase : RoomDatabase() {
   abstract fun contactDao(): ContactDao

   abstract fun templateDao(): TemplateDao

   abstract fun eventDao(): EventDao
}
